styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,371 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
```

Comment:
Any nesting and other code that becomes necessary due to overriding bootstrap, should go into the `styleguide/_themes/global/scss/bootstrap_overrides.scss` file.

That file is exempt from sass linting so we can get away with doing that kind of stuff there. That said, that file should be used sparingly, and only for properties that must be that specific to override. So if there are any properties you don't need that level of specificity for, place them in a top-level class instead. 

Additionally, if you make use of the bootstrap override file in a component, add a comment at the top of that file indicating some property overrides exist in the separate file to override bootstrap-specific styles, to assist with future maintainability.